### DISCUSSION TITLE
Customize body of webhook contact point in alerting

### DISCUSSION THREAD

**Discussion by Helloworld-can** [grafana/grafana#68158] [Alerting]
Hello,

Was hoping if there's  way we can achieve this use case within Grafana.

Idea is to trigger Webhook(with Body) event whenever a panel goes in Alerting state.

Was hoping if anyone has been able to achieve this?

Type: Grafana On-Prem
version: 9.2.1
Alerting Type: Unified Alerting
Alert Manager: Grafana

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82696. Feel free to continue the discussion around this there. Thank you!
